pallets/automation-time/src/tests_calculation.rs

```diff
@@ -1,213 +0,0 @@
-	SiblingParachainAsNative<cumulus_pallet_xcm::Origin, Origin>,
-	SignedAccountId32AsNative<RelayNetwork, Origin>,
-	XcmPassthrough<Origin>,
-);
-
-impl pallet_timestamp::Config for Test {
-	type Moment = u64;
-	type OnTimestampSet = ();
-	type MinimumPeriod = MinimumPeriod;
-	type WeightInfo = ();
-}
-
-parameter_types! {
-	pub const MaxScheduleSeconds: u64 = u64::MAX;
-	pub RelayChainOrigin: Origin = cumulus_pallet_xcm::Origin::Relay.into();
-}
-
-impl pallet_automation_time::Config for Test {
-	type Event = Event;
-	type MaxTasksPerSlot = MaxTasksPerSlot;
-	type MaxExecutionTimes = MaxExecutionTimes;
-	type MaxScheduleSeconds = MaxScheduleSeconds;
-	type MaxBlockWeight = MaxBlockWeight;
-	type MaxWeightPercentage = MaxWeightPercentage;
-	type UpdateQueueRatio = UpdateQueueRatio;
-	type SecondsPerBlock = SecondsPerBlock;
-	type WeightInfo = MockWeight<Test>;
-	type ExecutionWeightFee = ExecutionWeightFee;
-	type NativeTokenExchange = CurrencyAdapter<Balances, DealWithExecutionFees<Test>>;
-	type Origin = Origin;
-	type XcmSender = TestSendXcm;
-	type DelegatorActions = MockDelegatorActions<Test, Balances>;
-}
-
-pub struct XcmConfig;
-impl xcm_executor::Config for XcmConfig {
-	type Call = Call;
-	type XcmSender = TestSendXcm;
-	type AssetTransactor = DummyAssetTransactor;
-	type OriginConverter = XcmOriginToCallOrigin;
-	type IsReserve = ();
-	type IsTeleporter = ();
-	type LocationInverter = InvertNothing;
-	type Barrier = Barrier;
-	type Weigher = FixedWeightBounds<UnitWeightCost, Call, MaxInstructions>;
-	type Trader = DummyWeightTrader;
-	type ResponseHandler = ();
-	type AssetTrap = XcmPallet;
-	type AssetClaims = XcmPallet;
-	type SubscriptionService = XcmPallet;
-}
-
-impl pallet_xcm::Config for Test {
-	type Event = Event;
-	type SendXcmOrigin = EnsureXcmOrigin<Origin, LocalOriginToLocation>;
-	type XcmRouter = (TestSendXcm, TestSendXcm);
-	type LocationInverter = InvertNothing;
-	type ExecuteXcmOrigin = EnsureXcmOrigin<Origin, LocalOriginToLocation>;
-	type XcmExecuteFilter = Everything;
-	type XcmExecutor = XcmExecutor<XcmConfig>;
-	type XcmTeleportFilter = Everything;
-	type Weigher = FixedWeightBounds<UnitWeightCost, Call, MaxInstructions>;
-	type XcmReserveTransferFilter = Everything;
-	type Origin = Origin;
-	type Call = Call;
-	const VERSION_DISCOVERY_QUEUE_SIZE: u32 = 100;
-	type AdvertisedXcmVersion = AdvertisedXcmVersion;
-}
-
-impl cumulus_pallet_xcm::Config for Test {
-	type Event = Event;
-	type XcmExecutor = XcmExecutor<XcmConfig>;
-}
-
-// Build genesis storage according to the mock runtime.
-fn new_test_ext(state_block_time: u64) -> sp_io::TestExternalities {
-	let t = system::GenesisConfig::default().build_storage::<Test>().unwrap();
-	let mut ext = sp_io::TestExternalities::new(t);
-	ext.execute_with(|| System::set_block_number(1));
-	ext.execute_with(|| Timestamp::set_timestamp(state_block_time));
-	ext
-}
-
-#[test]
-fn validate_and_schedule_task_arithmetic_overflow() {
```

Comment:
This test is not providing much value while requiring a decent amount of upkeep when the pallet changes (due to the amount of repeated code in this file). I've opted to just remove it. Thoughts?